package.json

```diff
@@ -5,7 +5,7 @@
   "author": "pmedianetwork",
   "main": "index.js",
   "scripts": {
-    "test": "jest --watch",
+    "test": "jest",
```

Comment:
@adamgajzlerowicz lets add the :watch script separtly please:

`"test:watch": "npm run test -- --watch"`

Reply:
Sure 
https://github.com/pmedianetwork/eslint-config/pull/79/commits/762cc3f5052d69256d57a7711e31c37c21aad591